nebula_common/include/nebula_common/continental/continental_ars548.hpp

```diff
@@ -292,6 +305,74 @@ constexpr int blockage_test_ongoing = 2;
 constexpr int min_odometry_hz = 10;
 constexpr int max_odometry_hz = 50;
 
+struct FieldInfo
+{
+  FieldInfo(
+    bool min_value_available, bool max_value_available, bool resolution_available, float min_value,
+    float max_value, float resolution)
+  : min_value_available(min_value_available),
+    max_value_available(max_value_available),
+    resolution_available(resolution_available),
+    min_value(min_value),
+    max_value(max_value),
+    resolution(resolution)
+  {
+  }
+  bool min_value_available;
+  bool max_value_available;
+  bool resolution_available;
+  float min_value;
+  float max_value;
+  float resolution;
+};
+
```

Comment:
Similar concerns to above

https://github.com/tier4/nebula/pull/284#discussion_r2038816877

I agree that since the info fields are a property of the sensor, they should probably not be put into `nebula_ros`.
So, maybe letting ROS into nebula_common is inevitable after all...

Reply:
If this is left here, this would be a nice opportunity to combine the `bool`/`float` pairs into `std::optional`s for brevity below

Reply:
Addressed in https://github.com/tier4/nebula/pull/284/commits/bd9f1640d48d39fe3b28a7d7950a2e5dc2a8e9f1